src/TodoApp/Features/Todos/Pages/TodosPage.xaml.cs

```diff
@@ -8,28 +7,34 @@ public partial class TodosPage
         public TodosPage()
         {
             InitializeComponent();
-            TodoContentView.OnMenuClicked += Menu_Clicked;
-            SideBarView.OnBackClicked += Back_Clicked;
         }
 
-        private void Menu_Clicked()
+        protected override void OnAppearing()
         {
-            var vm = BindingContext as TodosPageViewModel;
-            vm.SidebarMenuVisible = !vm.SidebarMenuVisible;
+            base.OnAppearing();
 
-            TodoContentView.TranslateTo(App.Current.MainPage.Width * 0.8, 0, 500, Easing.CubicIn);
-            TodoContentView.ScaleTo(0.85, 500, Easing.CubicIn);
-            SideBarView.TranslateTo(0, 0, 500, Easing.CubicIn);
+            TodoContentView.OnMenuClicked += Menu_ClickedEvent;
```

Comment:
By creating a new instance of EventHandler to wrap the method, it always uses WeakReference to make things loose coupling. We don't need to add and remove this way.